src/connectors/sort-by/connectSortBy.js

```diff
@@ -164,6 +165,7 @@ export default function connectSortBy(renderFn, unmountFn = noop) {
         return {
           currentRefinement: helper.state.index,
           options: transformItems(items),
+          canRefine: Boolean(results && results.nbHits > 0),
```

Comment:
actually, no, this is wrong. You can still change index if this index has no refinements, so canRefine is probably always true

Reply:
You're right. If it's always going to be true, then I think we shouldn't add it just for the consistency. What do you think?

Reply:
I'm not sure, I think we should probably leave it out if there's no reasonable for it not to refine, same as DX-1312

Reply:
But you're right, I don't since it's worth adding at all.

One use case for `canRefine === false` I think might be when we have only one index and it's already selected, and no indices at all. Both have workaround maybe `canRefine` superfluous IMHO.